lib/index.js

```diff
@@ -100,6 +119,34 @@ export default {
     })
   },
 
+  async getVersion(configFile: string, executable: string): Promise<?string> {
+    const dir = Path.dirname(configFile)
+    const server = servers.get(dir)
+    if (server && server.version != null) {
+      return server.version
+    }
+
+    const p = exec(executable, ['version', '--json'], {
```

Comment:
You can `await` for this instead of using `.then`

Promises and async/await are interchangable in Javascript

Reply:
`p` is stored in the server variable in order to avoid duplicate calls to get the version... I'm not sure how this could work with `await`...

Reply:
Fair enough 👍